src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -28,17 +28,29 @@
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type;
+
 @Data
 @SuperBuilder(toBuilder = true)
 @NoArgsConstructor
+@JsonTypeInfo(
+    use = JsonTypeInfo.Id.NAME,
+    include = JsonTypeInfo.As.EXISTING_PROPERTY, 
+    property = "model",
+    visible = true)
+@JsonSubTypes({
+  @Type(value = FhirMeasure.class, name = "QI-Core v4.1.1"),
+  @Type(value = QdmMeasure.class, name = "QDM v5.6")
```

Comment:
I don't know if we are going to bring over QDM 5.5 measures but in case we are then assuming we'll add one more subtype for it.

Reply:
Ok so should I add it now? @adongare

Reply:
I'm not sure either. Probably we can add it later if needed.